Spi master spec

XdoctorwhoZ opened the pull request.

Commit:
specification spi master

Commit:
organise folder

Commit:
modify pinout

Commit:
req usb protocole

Commit:
add status

XdoctorwhoZ commented on projects/template/README.md:
Pourquoi tu as supprimé le template ?

MageTomcatF14 replied:
J'ai remis le template au bon endroit.

XdoctorwhoZ commented on projects/spi/README.md:
```diff
@@ -0,0 +1,59 @@
+#### [REQ_4000] Start of the program
+**The internal led must be on, during the start and the excecution of the firmware**
+
+#### [REQ_5000] End of the program
```
bizarre cette exigence ? le programme s'arréte quand la carte est débranchée donc la led s'éteint. Pourquoi vous avez tous mis ça ?

XdoctorwhoZ commented on projects/spi/README.md:
```diff
@@ -0,0 +1,59 @@
+**The firmware must still be running when the spi connections are disconnected**
+
+#### [REQ_7000] clock frequency
+**When the firmware starts, the GPIO 6 must deliver 125KHz**
```
??? quand le firmware demarre tu envoies une clock de 125Khz sur le GPIO6 ? je comprends pas pourquoi ?

MageTomcatF14 replied:
J'ai mal formulé l'exigence. Je voulais dire que la fréquence de fonctionnement du SPI est de 125KHz.

fdmysterious replied:
D'où sort cette fréquence ? A priori, on souhaiterais au moins pouvoir régler la fréquence du SPI.

XdoctorwhoZ reviewed (commented):
Je ne comprend rien du tout à ta spécification.

Commit:
update REQ 5000 and add requirements on SPI polarity

Commit:
add function codes for SPI configuration and operations

Commit:
update

Commit:
modify REQ 6000

Commit:
restore template and modify REQ 10000

Commit:
create source file folder

Commit:
create src file

Helfezer committed:
Merge branch 'main' into spi_master_spec
To start from latest point before reworking spi master spec

Helfezer committed:
Update SPI spec

Update following UART spec exemple, section 1 and 2 keep without any modification

Helfezer committed:
Add development set-up schematic

Helfezer committed:
Add intermediate objectives spec

Helfezer committed:
Rename folder to spi_master

Helfezer committed:
Retrieve missing img
